cmd/bbolt/command_check.go

```diff
@@ -4,25 +4,36 @@ import (
 	"fmt"
 
 	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
 
 	bolt "go.etcd.io/bbolt"
 	"go.etcd.io/bbolt/internal/guts_cli"
 )
 
+type checkPageOptions struct {
+	pageId uint64
+}
+
+func (o *checkPageOptions) AddFlags(fs *pflag.FlagSet) {
+	fs.Uint64VarP(&o.pageId, "pageId", "", o.pageId, "check db integrity starting from the given pageId")
+}
+
 func newCheckCommand() *cobra.Command {
+	var o checkPageOptions
 	checkCmd := &cobra.Command{
 		Use:   "check <bbolt-file>",
 		Short: "verify integrity of bbolt database data",
-		Args:  cobra.ExactArgs(1),
+		Args:  cobra.MinimumNArgs(1),
```

Comment:
I don't think we need to change this. The `--pageId` is just an option instead of an argument.

Reply:
👍🏽 fixed